controls/problems.js

```diff
@@ -166,4 +167,50 @@ helper.postIde = function (req, res) {
     });
 }
 
+helper.renderUsers = function (req, res) {
```

Comment:
Add a few comments in this function as it looks a bit messy :sweat_smile: . Also, the table should display the users who have solved 0 question and the users with the same number of questions solved should have the same rank.

Reply:
i have removed the code to fetch users and only the front end page is added in this PR.

Reply:
you can work on the rendering part @Vishal1541 after merging this... This removes unnecessary files and makes the ranking page similar to other pages

Reply:
@vsvipul sure :+1: